cmd/soroban-cli/src/commands/network/container/logs.rs

```diff
@@ -0,0 +1,52 @@
+use chrono::offset::Local;
+use futures_util::TryStreamExt;
+
+use crate::commands::network::container::shared::{
+    connect_to_docker, Error as ConnectionError, Network, DOCKER_HOST_HELP,
+};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    ConnectionError(#[from] ConnectionError),
+
+    #[error("⛔ ️Failed to tail container: {0}")]
+    TailContainerError(#[from] bollard::errors::Error),
+}
+
+#[derive(Debug, clap::Parser, Clone)]
+pub struct Cmd {
+    /// Network to tail
+    pub network: Network,
+
+    #[arg(short = 'd', long, help = DOCKER_HOST_HELP, env = "DOCKER_HOST")]
+    pub docker_host: Option<String>,
+}
+
+impl Cmd {
+    pub async fn run(&self) -> Result<(), Error> {
+        let container_name = format!("stellar-{}", self.network);
+        println!("ℹ️  Tailing logs for {container_name}");
```

Comment:
Since this output is not the logs, I would either remove it, or send it to stderr at least, so that stdout isn't impaired by it. But, given that it looks like we actually log out the stderr and stdout, it might be important we don't add any of our own logs to the output either.
```suggestion
        eprintln!("ℹ️  Tailing logs for {container_name}");
```
or

```suggestion
```

Reply:
Good point! I'll remove it.